packages/model/src/user.ts

```diff
@@ -10,6 +10,8 @@ export interface Invitee {
   department?: string;
   institution?: string;
   location?: string;
+  avatarURL?: string;
+  skills?: string[];
```

Comment:
We don't expect skills when adding a new user. The data is changed on the CMS.

Reply:
I wanted to add all user props to the create user method so that I could test all of them return. I will address this on another PR. 👍